aiohttp_devtools/runserver/watch.py

```diff
@@ -119,10 +121,33 @@ def _start_dev_server(self) -> None:
         self._process = Process(target=serve_main_app, args=(self._config, tty_path))
         self._process.start()
 
-    def _stop_dev_server(self) -> None:
+    async def _stop_dev_server(self) -> None:
         if self._process.is_alive():
             logger.debug('stopping server process...')
+            if self._config.shutdown_endpoint:  # a workaround for singals not working on Windows
+                url = 'http://localhost:{}{}/shutdown'.format(self._config.main_port, self._config.path_prefix)
+                logger.debug('attempting to stop process via shutdown endpoint {}'.format(url))
+                try:
+                    # these errors are expected because the request kills the server *immediately*
+                    with suppress(ServerDisconnectedError, ClientConnectorError):
```

Comment:
Possibly not needed now with the call_soon() (and certainly not once aiohttp 3.9 is out, as we allow handlers to finish before shutdown completes).

Reply:
I tested with both `call_soon` and `call_later` and the `suppress` is still necessary.